src/test/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesAppDeployerIntegrationIT.java

```diff
@@ -925,6 +926,47 @@ public void createStatefulSetWithOverridingRequest() throws Exception {
 		});
 	}
 
+	@Test
+	public void createStatefulSetWithPVCDefaultName() throws Exception {
+		Map<String, String> props = new HashMap<>();
+		props.put(KubernetesAppDeployer.COUNT_PROPERTY_KEY, "3");
+		props.put(KubernetesAppDeployer.INDEXED_PROPERTY_KEY, "true");
+		AppDefinition definition = new AppDefinition(randomName(), null);
+		AppDeploymentRequest appDeploymentRequest = new AppDeploymentRequest(definition, testApplication(), props);
+
+		KubernetesAppDeployer deployer = new KubernetesAppDeployer(new KubernetesDeployerProperties(),
+				kubernetesClient);
+
+		log.info("Deploying {}...", appDeploymentRequest.getDefinition().getName());
+		String deploymentId = deployer.deploy(appDeploymentRequest);
+		Timeout timeout = deploymentTimeout();
+		await().pollInterval(Duration.ofMillis(timeout.pause))
+				.atMost(Duration.ofMillis(timeout.maxAttempts * timeout.pause))
+				.untilAsserted(() -> {
+					assertThat(appDeployer().status(deploymentId).getState()).isEqualTo(DeploymentState.deployed);
+				});
+
+		Map<String, String> selector = Collections.singletonMap(SPRING_APP_KEY, deploymentId);
```

Comment:
I get a Compilation failure of missing `SPRING_APP_KEY`

Reply:
Rebase would fix this. Will update it.